_includes/templates/install/cassandra-rhel-install.md

```diff
@@ -1,5 +1,45 @@
-Please follow [official Apache Cassandra installation guide](https://cassandra.apache.org/doc/latest/cassandra/installing/installing.html#install-as-rpm-package).
+Create and edit the Cassandra repository file:
 
-You can use Astra DB cloud instead installing your own Cassandra.
+```bash
+sudo nano /etc/yum.repos.d/cassandra.repo
+```
+{: .copy-code}
+
+Paste the following content into the file:
+
+```ini
+[cassandra]
+name=Apache Cassandra
+baseurl=https://redhat.cassandra.apache.org/50x/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://downloads.apache.org/cassandra/KEYS
+```
+{: .copy-code}
+
+The following commands will install Apache Cassandra and its command-line tools; confirm GPG key imports if prompted:
+
+```bash
+# Install packages
+sudo dnf update
+sudo dnf install cassandra cassandra-tools
+# Start service
+sudo systemctl daemon-reload
+sudo service cassandra start
+# Optional: Configure Cassandra to start on boot
```

Comment:
it is hard to reason why Cassandra should not start on boot by default. I suggest to remove the word "Optional"